src/ng/compile.js

```diff
@@ -2245,43 +2338,66 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
               directiveNormalize(nodeName), 'E', maxPriority, ignoreDirective);
 
           // iterate over the attributes
-          for (var attr, name, nName, ngAttrName, value, isNgAttr, nAttrs = node.attributes,
+          for (var attr, name, nName, value, ngPrefixMatch, nAttrs = node.attributes,
                    j = 0, jj = nAttrs && nAttrs.length; j < jj; j++) {
             var attrStartName = false;
             var attrEndName = false;
 
+            var isNgAttr = false, isNgProp = false, isNgEvent = false;
+            var multiElementMatch;
+
             attr = nAttrs[j];
             name = attr.name;
             value = attr.value;
 
-            // support ngAttr attribute binding
-            ngAttrName = directiveNormalize(name);
-            isNgAttr = NG_ATTR_BINDING.test(ngAttrName);
-            if (isNgAttr) {
+            name = name.toLowerCase();
```

Comment:
This is an interesting one. Note that previously `name` did not have the `.toLowerCase()` assigned to it, we only did:
```
nName = directiveNormalize(name.toLowerCase());
```
Also... `attr.name` should always be lower case anyway, since browsers normalize attribute names to all lower case. However without this line one test failed in IE9: https://travis-ci.org/angular/angular.js/jobs/406368530#L687. An alternative would be making `EVENT_HANDLER_ATTR_REGEXP` case-insensitive, which I'm tempted to do either way...

Another weird one... if the `.toLowerCase()` is done above as `name = attr.name.toLowerCase()` then a lot of animation tests fail. (I seem to recall something about SVG attributes being different? Didn't check if the tests used SVG though...).

Reply:
TODO: move this into the event/prop specific code to avoid unnecessary changes